src/components/message.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+interface UserMessage {
+  message: string;
+}
+
+const Message = (props: UserMessage): any => {
```

Comment:
Instead of returning `any`, it should return `JSX.Element`.

It's good to use `any`, though, as you are learning. Eventually, we'll want to get rid of all the `any` usages.

Reply:
Writing this down, thanks!